docs/user/commandline_tools.rst

```diff
@@ -27,31 +27,33 @@ Run ``geomesa`` without any arguments to produce the following usage text::
     $ geomesa
     Usage: geomesa [command] [command options]
       Commands:
-        add-index           Add or update indices for an existing GeoMesa feature type
-        create              Create a GeoMesa feature type
-        deletecatalog       Delete a GeoMesa catalog completely (and all features in it)
-        deletefeatures      Delete features from a table in GeoMesa. Does not delete any tables or schema information.
-        deleteraster        Delete a GeoMesa Raster table
-        describe            Describe the attributes of a given GeoMesa feature type
-        env                 Examine the current GeoMesa environment
-        explain             Explain how a GeoMesa query will be executed
-        export              Export features from a GeoMesa data store
-        genavroschema       Generate an Avro schema from a SimpleFeatureType
-        getsft              Get the SimpleFeatureType of a feature
-        help                Show help
-        ingest              Ingest/convert various file formats into GeoMesa
-        ingestraster        Ingest raster files into GeoMesa
-        keywords            Add/Remove/List keywords on an existing schema
-        list                List GeoMesa feature types for a given catalog
-        queryrasterstats    Export queries and statistics about the last X number of queries to a CSV file.
-        removeschema        Remove a schema and associated features from a GeoMesa catalog
-        stats-analyze       Analyze statistics on a GeoMesa feature type
-        stats-bounds        View or calculate bounds on attributes in a GeoMesa feature type
-        stats-count         Estimate or calculate feature counts in a GeoMesa feature type
-        stats-histogram     View or calculate counts of attribute in a GeoMesa feature type, grouped by sorted values
-        stats-top-k         Enumerate the most frequent values in a GeoMesa feature type
-        tableconf           Perform table configuration operations
-        version             Display the installed GeoMesa version
+        add-attribute-index    Run a Hadoop map reduce job to add an index for attributes
```

Comment:
can you re-run this now that you've merged master? some of the descriptions got tweaked (e.g. `create - Create a GeoMesa feature type` vs `create - Create a feature definition in a GeoMesa catalog`